docs/JestObjectAPI.md

```diff
@@ -404,12 +415,24 @@ Example:
 jest.setTimeout(1000); // 1 second
 ```
 
+### `jest.useFakePromises()`
```

Comment:
this should note that `async-await` (as long as it's not transpiled into generators and normal `Promise` with Babel or something like it) will not work